src/Documentation/fpga/xrt.rst

```diff
@@ -251,17 +264,18 @@ Driver Interfaces
 xmgmt Driver Ioctls
 -------------------
 
-Ioctls exposed by xmgmt driver to user space are enumerated in the following table:
+Ioctls exposed by xmgmt driver to user space are enumerated in the following
+table:
 
-== ===================== ============================= ===========================
+== ===================== ============================ ==========================
 #  Functionality         ioctl request code            data format
-== ===================== ============================= ===========================
+== ===================== ============================ ==========================
 1  FPGA image download   XMGMT_IOCICAPDOWNLOAD_AXLF    xmgmt_ioc_bitstream_axlf
 2  CL frequency scaling  XMGMT_IOCFREQSCALE            xmgmt_ioc_freqscaling
```

Comment:
We actually did not implement XMGMT_IOCFREQSCALE. Wondering if we should remove this from our doc and header?

Reply:
Point taken, will update the PR

Reply:
done

Reply:
Thanks. I'll file a PR to remove it from our header now.